src/components/Header/Header.tsx

```diff
@@ -21,15 +39,119 @@ export interface HeaderProps {
 }
 
 export const Header: React.FC<HeaderProps> = ({
+  behavior,
   className,
   children,
+  pinnedStop = 152,
```

Comment:
@ifrost1 Not sure why 152 specifically, but if it's not needing that level of accuracy may I suggest a rounder number?
```suggestion
  pinnedStop = 150,
```